apps/server/src/shared/infra/oauth-provider/hydra/hydra.service.spec.ts

```diff
@@ -155,8 +159,22 @@ describe('HydraService', () => {
 
 	describe('Logout Flow', () => {
 		describe('acceptLogoutRequest', () => {
-			it('should throw', () => {
-				expect(() => service.acceptLogoutRequest('')).toThrow(NotImplementedException);
+			it('should make http request', async () => {
+				// Arrange
+				const response: RedirectResponse = { redirect_to: 'redirect_mock' };
+				httpService.request.mockReturnValue(of(createAxiosResponse(response)));
+				const config: AxiosRequestConfig = {
+					method: 'PUT',
+					url: `${hydraUri}/oauth2/auth/requests/logout/accept?logout_challenge=challenge_mock`,
+					headers: { 'X-Forwarded-Proto': 'https' },
+				};
+
+				// Act
+				const redirect: RedirectResponse = await service.acceptLogoutRequest('challenge_mock');
+
+				// Assert
+				expect(httpService.request).toHaveBeenCalledWith(expect.objectContaining(config));
+				expect(redirect).toEqual(response);
```

Comment:
minor thing, but before reading the arrange part I got confused which is the redirect and which the response.
Maybe it makes sense to call them either `redirect` and `mockRedirect` or `response` and `mockResponse`

Reply:
changed it to responseMock and response ;)